src/Avalonia.Diagnostics/Diagnostics/DevTools.cs

```diff
@@ -5,20 +5,28 @@
 using Avalonia.Diagnostics.Views;
 using Avalonia.Input;
 using Avalonia.Interactivity;
-
+#nullable enable 
 namespace Avalonia.Diagnostics
 {
     public static class DevTools
```

Comment:
Why it is public? Can i make internal?

Reply:
Do you mean why `DevTools` class is `public`? In this case - well this is an API for the user to attach it, I think there is no other available way.

Reply:
There is DevToolsExtensions

Reply:
Oh, I've forgotten about that extension - in this case we might want to make this part internal to reduce API surface (if both provide the same functionality).